### PR TITLE
rename bindEntityEvents->bindEvents

### PR DESCRIPTION
**Pull request by denar90**
### Proposed changes
 - bindEntityEvents became bindEvents, 
 - bindRadioRequests became bindRequests

Link to the issue:
#2989 

Also maybe we should think about removing `bindRadioEvents`, `unbindRadioEvents` because they are identical to `bindEntityEvents/unbindEntityEvents`?  What do you think guys?

### CODE REVIEW

**Comment by paulfalgout** [comment]
Oh yeah.. I think if we're doing this bindRadioEvents doesn't need to exist.  can just use `bindEvents`

File: src/bind-events.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ function iterateEvents(target, entity, bindings, actionName) {
   if (!_.isObject(bindings)) {
     throw new MarionetteError({
       message: 'Bindings must be an object.',
-      url: 'marionette.functions.html#marionettebindentityevents'
+      url: 'marionette.functions.html#marionettebindEvents'
```
Comment (paulfalgout) [inline src/bind-events.js]: I don't think the url will have an uppercase E

File: test/unit/bind-radio-request.spec.js [hunk 2]
```diff
@@ -76,14 +76,14 @@ describe('Marionette.bindRadioRequests', function() {
...
-        Marionette.bindRadioRequests(this.target, this.channel, 'replyFooStub');
+        Marionette.bindRequests(this.target, this.channel, 'replyFooStub');
       }.bind(this);
     });
 
     it('should error', function() {
       expect(this.run).to.throw(Marionette.Error, new Marionette.Error({
         message: 'Bindings must be an object.',
-        url: 'marionette.functions.html#marionettebindradiorequests'
+        url: 'marionette.functions.html#marionettebindRequests'
```
Comment (paulfalgout) [inline test/unit/bind-radio-request.spec.js]: again these'll match links on the website that are generated from the `docs/*.md` and so I think those `#hash` are all lower case.

That does point out something though.. the docs need to be updated as well to reflect these changes.

Comment (denar90) [inline test/unit/bind-radio-request.spec.js]: What do you think if I change doc for `bindEvents` but for others in scope of task #3055 ?

File: src/backbone.marionette.js [hunk 3]
```diff
@@ -58,12 +58,12 @@ Marionette.noConflict = function() {
...
 // Utilities
-Marionette.bindEntityEvents = proxy(bindEntityEvents);
-Marionette.unbindEntityEvents = proxy(unbindEntityEvents);
-Marionette.bindRadioEvents = proxy(bindEntityEvents);
-Marionette.unbindRadioEvents = proxy(unbindEntityEvents);
-Marionette.bindRadioRequests = proxy(bindRadioRequests);
-Marionette.unbindRadioRequests = proxy(unbindRadioRequests);
+Marionette.bindEvents = proxy(bindEvents);
+Marionette.unbindEvents = proxy(unbindEvents);
+Marionette.bindRadioEvents = proxy(bindEvents);
```
Comment (paulfalgout) [inline src/backbone.marionette.js]: I think it is perfectly ok to remove `bindRadioEvents` here as it matches the renaming of `bindRadioRequests`

**Comment by paulfalgout** [comment]
👍  I personally like how this turned out a lot.  And it reduces the v3 API which is a big plus.

**Comment by JSteunou** [comment]
Totally in, that's a great simplification! :+1:

**Pull request merged**